src/components/Navigation.js

```diff
@@ -0,0 +1,14 @@
+import { FaChevronLeft } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+
+export const NavBar = () => (
+  <>
+    <div>
+      <Link to="/" className="home-btn">
+        <FaChevronLeft />
+      </Link>
+    </div>
+  </>
```

Comment:
- [ ] Like the previous reviewer mentioned, kindly ensure this section matches the design given
### Your design
![](https://user-images.githubusercontent.com/77106962/227536645-e4268e17-583e-410e-a30e-1bbbb2c0e737.png)

### The given design
![](https://user-images.githubusercontent.com/77106962/226725061-a0c076f9-182f-4800-998e-32b186d21cd7.png)